graphics/mesa/surfaceflinger.te

```diff
@@ -26,3 +26,4 @@ allow surfaceflinger hal_graphics_composer_default:file read;
 allow surfaceflinger gpu_device:dir r_dir_perms;
 allow surfaceflinger sysfs_app_readable:file r_file_perms;
 allow surfaceflinger self:process execmem;
+allow surfaceflinger hal_graphics_composer_default:file { getattr open };
```

Comment:
please merge this line with line 25 (allow surfaceflinger hal_graphics_composer_default:file read;)